assets/containers/MapConnector.js

```diff
@@ -180,24 +179,16 @@ class MapConnector extends React.Component {
         }
     }
 
-    addBoundaryToMap() {
-        this.boundaryLayers = this.boundaryData.features.map(feature => (
-            L.geoJson(feature, {
-                style: {
-                    color: '#006',
-                    opacity: .7,
-                    weight: 2,
-                    fill: false
-                }
-            }).addTo(this.map)
-        ))
+    addBoundaryToMap(region) {
+        regionsBoundaries.setStyle(f => {
```

Comment:
This can be simplified a bit (I think):

```javascript
regionBoundaries.setStyle(f => f.properties.region === region ? {opacity: 1} : undefined)
```